sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -447,12 +348,7 @@ public void SetVariable(string variableName, string value, Func<string, string>
             }
         }
 
-        public void DisableIdReuse()
-        {
-            _previousSession = null;
-        }
-
-        public bool HasRequests => _sessionInternal?.Entries.Count > 0;
+        public bool HasRequests { get; internal set; }
```

Comment:
Is there no longer any local way to calculate that the session has requests besides when we first call `CreateRequest`?

Reply:
Nope, is there a reason you would need to be able to calculate this?